### ISSUE TITLE
Overplotting After Create New Plots

### ISSUE THREAD

**Issue by sebastianfelipe** [madMAx43v3r/chia-plotter#823]
I didn't know how to name this, but I think is a near name. The main issue consists when n > 1. That is when you need to instantiate more than one plot. When the process arrives at the last stage, the current plot needs to be copied to the final directory, but at the same time, it begins to instantiate the next plot. That fails when you have the exact amount of necessary space to plot. And also, it takes too much time, more than the time needed to wait and then instantiate, is a lot more than that, because it has to read and write at the same time. It increases a lot the plotting time. So, when you do sequential plotting, it should wait until the plot is fully copied to the final directory, before start with the creation of a new one.

The other issue I have found, Is that the plot is not copied from one directory to another, is moved to it, so, when it is an issue (like I have experimented), you lose the opportunity to copy it manually (this occurs for n == 1 also). It would be very useful to have the copy operation and then remove everything at once.

Both of this issues can be optimized with those solutions. I'd really appreciate it would be fixed or configured in some way.

Thanks!
Sebastián.

**Comment by ataa** [comment]
`-w, --waitforcopy    Wait for copy to start next plot`

**Comment by sebastianfelipe** [comment]
> `-w, --waitforcopy Wait for copy to start next plot`

You're right. Didn't read that in the other version. Thanks! Ans about the second thing. Is it moving or copying? Because I had that issue and I lost the plot, after the creation and in the middle of the "copy".

**Comment by ataa** [comment]
Last time I was monitoring it on Ubuntu, It started with copy the final plot with tmp extension to the destination path, then it renamed it to .plot. But I believe if your -d path is the same as temp, it will move/rename it.
